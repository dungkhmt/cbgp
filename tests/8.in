50 50
50 80
14 3
49 42
36 15
19 11
27 6
47 17
47 2
21 5
28 7
34 29
32 26
37 4
20 5
29 14
4 5
32 13
43 0
6 22
49 36
18 20
43 10
32 10
6 2
24 0
41 6
28 1
36 41
48 24
32 25
45 4
30 9
20 23
38 6
24 7
35 45
49 39
9 4
9 12
15 0
25 9
5 8
26 11
3 0
40 15
34 16
46 39
44 26
32 33
31 9
47 7
37 45
27 28
32 11
29 3
24 2
2 3
35 37
34 14
32 9
27 1
22 14
3 16
13 10
42 18
18 5
17 7
42 5
11 16
42 15
12 0
43 19
30 23
27 36
36 1
26 4
32 31
11 3
43 33
18 23
6 3
1
30 5 71 2 60 3 57 6 29 21 4 4 63 11 66 2 22 22 38 13 36 8 48 7 36 11 37 9 52 11 26 1 49 8 64 3 18 5 37 7 15 6 0 0 57 16 16 7 33 2 39 11 31 23 70 4 68 2 53 9 29 20 39 14 45 8 37 10 50 9 25 22 76 0 27 22 42 32 4 0 28 2 64 12 22 1 34 7 40 32 25 25 2 0 63 5 30 2 21 0