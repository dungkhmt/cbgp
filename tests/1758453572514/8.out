(14, 3), (49, 42), (36, 15), (19, 11), (27, 6), (47, 17), (47, 2), (21, 5), (28, 7), (34, 29), (32, 26), (37, 4), (20, 5), (29, 14), (4, 5), (32, 13), (43, 0), (6, 22), (49, 36), (18, 20), (43, 10), (32, 10), (6, 2), (24, 0), (41, 6), (28, 1), (36, 41), (48, 24), (32, 25), (45, 4), (30, 9), (20, 23), (38, 6), (24, 7), (35, 45), (49, 39), (9, 4), (9, 12), (15, 0), (25, 9), (5, 8), (26, 11), (3, 0), (40, 15), (34, 16), (46, 39), (44, 26), (32, 33), (31, 9), (47, 7), (37, 45), (27, 28), (32, 11), (29, 3), (24, 2), (2, 3), (35, 37), (34, 14), (32, 9), (27, 1), (22, 14), (3, 16), (13, 10), (42, 18), (18, 5), (17, 7), (42, 5), (11, 16), (42, 15), (12, 0), (43, 19), (30, 23), (27, 36), (36, 1), (26, 4), (32, 31), (11, 3), (43, 33), (18, 23), (6, 3), 
numberIntersection=0 
minDistance=2.9285611805518585 
minAngle=0.029991004856878334 
sumAngle=56.55855202829693 
sumDistance=263518.16490119864 
minEdgeLength=3.0 
sumEdgeLength=2510.75986003316 

0.0000000000 2.9285611806 0.0299910049 56.5585520283 263518.1649011986 3.0000000000 2510.7598600332
0: (18, 54),
1: (4, 140),
2: (23, 114),
3: (26, 111),
4: (79, 54),
5: (9, 2),
6: (48, 124),
7: (4, 134),
8: (155, 0),
9: (65, 60),
10: (29, 69),
11: (28, 96),
12: (26, 55),
13: (34, 74),
14: (50, 114),
15: (3, 47),
16: (31, 100),
17: (8, 128),
18: (17, 45),
19: (26, 73),
20: (20, 20),
21: (0, 0),
22: (49, 119),
23: (25, 19),
24: (5, 78),
25: (48, 72),
26: (74, 96),
27: (13, 138),
28: (4, 137),
29: (42, 107),
30: (28, 22),
31: (52, 77),
32: (36, 88),
33: (38, 73),
34: (44, 99),
35: (190, 0),
36: (0, 147),
37: (84, 9),
38: (195, 200),
39: (0, 6),
40: (6, 52),
41: (36, 132),
42: (3, 44),
43: (25, 61),
44: (200, 200),
45: (200, 3),
46: (0, 3),
47: (11, 130),
48: (6, 55),
49: (0, 44),
18 54 4 140 23 114 26 111 79 54 9 2 48 124 4 134 155 0 65 60 29 69 28 96 26 55 34 74 50 114 3 47 31 100 8 128 17 45 26 73 20 20 0 0 49 119 25 19 5 78 48 72 74 96 13 138 4 137 42 107 28 22 52 77 36 88 38 73 44 99 190 0 0 147 84 9 195 200 0 6 6 52 36 132 3 44 25 61 200 200 200 3 0 3 11 130 6 55 0 44 
0: (18, 54), 1: (4, 140), 2: (23, 114), 3: (26, 111), 4: (79, 54), 5: (9, 2), 6: (48, 124), 7: (4, 134), 8: (155, 0), 9: (65, 60), 10: (29, 69), 11: (28, 96), 12: (26, 55), 13: (34, 74), 14: (50, 114), 15: (3, 47), 16: (31, 100), 17: (8, 128), 18: (17, 45), 19: (26, 73), 20: (20, 20), 21: (0, 0), 22: (49, 119), 23: (25, 19), 24: (5, 78), 25: (48, 72), 26: (74, 96), 27: (13, 138), 28: (4, 137), 29: (42, 107), 30: (28, 22), 31: (52, 77), 32: (36, 88), 33: (38, 73), 34: (44, 99), 35: (190, 0), 36: (0, 147), 37: (84, 9), 38: (195, 200), 39: (0, 6), 40: (6, 52), 41: (36, 132), 42: (3, 44), 43: (25, 61), 44: (200, 200), 45: (200, 3), 46: (0, 3), 47: (11, 130), 48: (6, 55), 49: (0, 44), 
numberIntersection=0 minAngle=0.029991004856878334 sumAngle=56.55855202812991 minEdgeLength=3.0 sumEdgeLength=2510.7598600331467 minDistance=2.9285611805518585 sumDistance=263518.16490116186 
Time taken: 184.523 s
